src/tools/spin/spinquic.cpp

```diff
@@ -791,8 +818,11 @@ main(int argc, char **argv)
         TryGetValue(argc, argv, "sessions", &SessionCount);
     }
 
-    uint32_t RngSeed = 6;
-    TryGetValue(argc, argv, "seed", &RngSeed);
+    uint32_t RngSeed = 0;
+    if (!TryGetValue(argc, argv, "seed", &RngSeed)) {
+        CxPlatRandom(sizeof(RngSeed), &RngSeed);
+    }
+    printf("Using seed value: 0x%d\n", RngSeed);
```

Comment:
```suggestion
    printf("Using seed value: 0x%u\n", RngSeed);
```
Unsigned int print, not signed.